contracts/src/Rollup.sol

```diff
@@ -64,6 +64,34 @@ abstract contract RollupBase is
         __Ownable_init();
         __UUPSUpgradeable_init();
     }
+
+    /// @inheritdoc IRollup
+    function setConfirmationPeriod(uint256 newPeriod) public override onlyOwner {
+        confirmationPeriod = newPeriod;
+        emit ConfigurationChanged();
+    }
+
+    /// @inheritdoc IRollup
+    function setChallengePeriod(uint256 newPeriod) public override onlyOwner {
+        challengePeriod = newPeriod;
+        emit ConfigurationChanged();
+    }
+
+    /// @inheritdoc IRollup
+    function setMinimumAssertionPeriod(uint256 newPeriod) public override onlyOwner {
+        minimumAssertionPeriod = newPeriod;
+        emit ConfigurationChanged();
+    }
+
+    /// @inheritdoc IRollup
+    function setBaseStakeAmount(uint256 newAmount) public override onlyOwner {
+        if (newAmount > baseStakeAmount) {
+            revert IncreaseStake();
+        }
+
+        baseStakeAmount = newAmount;
+        emit ConfigurationChanged();
```

Comment:
This event does not help anyone much. Consider adding the `previousValue` and `changedValue` in the event.